src/getting-started.md

```diff
@@ -123,17 +123,17 @@ The compiler has a configuration file which contains a ton of settings. We will
 provide some recommendations here that should work for most, but [check out
 this chapter for more info][config].
 
-[config]: ./building/how-to-build-and-run.html#create-a-configtoml
+[config]: ./building/how-to-build-and-run.md#create-a-configtoml
 
 In the top level of the repo:
 
-```sh
+```console
 $ x.py setup
 ```
 
 This will walk you through an interactive setup for x.py that looks like this:
 
-```
+```console
```

Comment:
Can you please avoid these unrelated changes here? See also https://github.com/rust-lang/rustc-dev-guide/issues/515.

Reply:
Heh, I don't even remember making these changes 😆